mu-plugins/media-import.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * WP-CLI command to import the images from unsplash API.
```

Comment:
Can you please add plugin info similar to joe did here https://github.com/mukeshpanchal27/webp-quality-analysis/blob/trunk/mu-plugins/wp-cli-webp-analysis-command.php#L2